pallets/manta-pay/Cargo.toml

```diff
@@ -65,17 +65,18 @@ precompute-coins = [
   "anyhow",
   "indoc",
   "manta-accounting/test",
-  "manta-sdk/download",
+  "manta-pay/download",
+  "manta-pay/test",
   "manta-util/std",
-  "rand",
+  "rand_chacha",
```

Comment:
also, changing the RNG is not described in changelog or PR

Reply:
Will add in PR.